docs/ingestion/supervisor.md

```diff
@@ -182,12 +192,16 @@ The following example shows a supervisor spec with `lagBased` autoscaler:
 ```
 </details>
 
-### Tuning configuration
+### `tuningConfig`
+
+The `tuningConfig` controls various tuning parameters specific to each ingestion method. See native ingestion specification documentation on the [`tuningConfig`](../ingestion/ingestion-spec.md#tuningconfig) for more information.
```

Comment:
```suggestion
The `tuningConfig` controls various tuning parameters specific to each ingestion method. See native ingestion specification documentation on the [`tuningConfig`](../ingestion/ingestion-spec.md#tuningconfig) for more information.
```
Is this link correct (../ingestion/ingestion-spec.md#tuningconfig)? Shouldn't it be (ingestion/ingestion-spec.md#tuningconfig)?